lib/logstash/filters/csv.rb

```diff
@@ -91,5 +98,44 @@ def filter(event)
 
   end # def filter
 
+  private
+
+  def ignore_field?(index)
+    !@columns[index] && !@autogenerate_column_names
+  end
+
+  def should_convert?(field_name)
+    !@convert[field_name].nil?
+  end
+
+  def convert(field_name, value)
```

Comment:
is interesting I don't think we use the method to access the variable, but I agree we should avoid possible redefinitions, so I changed the name.